index.js

```diff
@@ -21,6 +23,11 @@ module.exports = function (source) {
     const options = typeof optionsConfig === 'object' ?
         utils.getOptions(this) : utils.parseQuery(this.query);
 
+    if(options !== "undefined" && options !== null){
+        if(options.verbose == true) {
```

Comment:
[sug] It will be better if we avoid nesting `if`s, something like: `if (options && options.verbose)` will better. WDYT?

Reply:
fixed